Minor cleanups to params, vars, and function names

linearcombination opened the pull request:
Minor changes. Integration tests vet these changes.

linearcombination committed:
Remove function parameter and some local vars that were not used

linearcombination committed:
Remove redundant HtmlContent instantiation

The functions that were being wrapped in the HtmlContent instantiation
already return an HtmlContent instance.

PurpleGuitar commented on backend/document/domain/assembly_strategies.py:
```diff
@@ -5060,10 +5055,6 @@ def assemble_tw_as_iterator_for_book_then_lang(
     # Sort resources by language
     key = lambda resource: resource.lang_code
     # FIXME Do we need to sort anything other than
```
Looks like this comment got cut in half -- I recommend restoring it or deleting if no longer needed.

linearcombination replied:
Good catch! Thank you! This led me to look a little closer and add a couple more integration tests.

linearcombination committed:
Remove vestigial half comment per PR review comment

linearcombination committed:
Minor code formatting via black

linearcombination committed:
Add integration tests for two document request cases

Case one: request bible commentary and translations words
Case two: request bible commentary and translations words for print,
i.e., compacct

linearcombination committed:
Automatically update some packages via make local-install-deps-dev
